Example/Tests/BPKColorTest.m

```diff
@@ -18,7 +18,7 @@
 
 #import <XCTest/XCTest.h>
 
-#import <Backpack/Backpack.h>
+#import <Backpack/Color.h>
```

Comment:
To confirm, this is the breaking change?

Reply:
Yeah